docs/source/custom/package.rst

```diff
@@ -93,7 +94,40 @@ keywords should mention ``cobald`` for findability.
         ...
     )
 
-There are currently no ``entry_points`` used by :py:mod:`cobald`.
+Configuration Plugins
+*********************
+
+Packages can declare callables as plugins for the YAML configuration format.
```

Comment:
What is the point here? Why should I do this?

Reply:
Should the "this allows using the extension ..." part be moved up to here?